packages/dates/src/parse-date-string.ts

```diff
@@ -3,49 +3,60 @@ import {applyTimeZoneOffset} from './apply-time-zone-offset';
 /**
  * Allowed date string formats
  * yyyy-mm-dd
- * yyyy-mm-ddThh:mm:ss
- * yyyy-mm-ddThh:mm:ss+hh:mm
- * yyyy-mm-ddThh:mm:ss-hh:mm
- * yyyy-mm-ddThh:mm:ssZ
+ * yyyy-mm-ddThh:mm:ss.fff
+ * yyyy-mm-ddThh:mm:ss.fff+hh:mm
+ * yyyy-mm-ddThh:mm:ss.fff-hh:mm
+ * yyyy-mm-ddThh:mm:ss.fffZ
  */
-const DATE_TIME_PARTS_REGEX = /^(\d{4})-(\d{2})-(\d{2})(?:T(\d{2}):(\d{2}):(\d{2})(?:(Z|(?:(\+|-)(\d{2}):(\d{2}))))?)?$/;
+const DATE_TIME_PARTS_REGEX = /^(\d{4})-(\d{2})-(\d{2})(?:T(\d{2}):(\d{2}):(\d{2})(?:\.(\d{1,3}))?(?:(Z|(?:(\+|-)(\d{2}):(\d{2}))))?)?$/;
 
-export function parseDateString(dateString: string, timeZone?: string) {
+export function parseDateStringParts(dateString: string) {
```

Comment:
are we planning to export this from the package, or is it only being exported for testing purposes?

Reply:
yea it is intended to be a package export.

Reply:
Ah, then we should add this to the README as well
Perhaps give it it's own file / test file

Reply:
yea sounds good!